src/Client/AzureManaged/DurableTaskSchedulerClientOptions.cs

```diff
@@ -6,8 +6,8 @@
 using Azure.Identity;
 using Grpc.Core;
 using Grpc.Net.Client;
-using Microsoft.DurableTask;
 using Microsoft.DurableTask.Client;
+using grpcConfig = Grpc.Net.Client.Configuration;
```

Comment:
nit: Let's use Pascal casing here, which is the standard for namespaces.
```suggestion
using GrpcConfig = Grpc.Net.Client.Configuration;
```